AppCenterLoaderApp/appcenter-loader/src/main/java/com/microsoft/appcenter/loader/StartupType.java

```diff
@@ -1,3 +1,7 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+//
```

Comment:
Why adding a blank line? On other repos this is not what we did. Did you use a tool that does that?
Do we have only 15 files on Unity that needs licence?

Reply:
A lot of files already have this header 
https://github.com/Microsoft/AppCenter-SDK-Unity/blob/develop/Assets/AppCenter/AppCenterBehavior.cs#L3

Reply:
It looks like a blank line copied from existing file headers

Reply:
Lets remove that blank line then.